docs/mobile/development.md

```diff
@@ -67,6 +67,12 @@ cd /path/to/mobile-courses-example-course
 mobile-teach check
 ```
 
+If you want to exclude warnings:
+
+```sh
+mobile-teach check | grep -v warning
```

Comment:
In the next release, it should be possible to do `mobile-teach check --no-warnings` (But I have not yet published this release)